<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>720</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Home â€” Menu Tabel</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
QLabel#lblTitle { font-size: 18pt; font-weight: 700; }
QLabel#lblSubtitle { color: #666; }
QPushButton { min-width: 160px; min-height: 60px; font-size: 11pt; }
QPushButton:hover { background: #eef2f7; }
    </string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>12</number>
    </property>
    <item>
     <widget class="QLabel" name="lblTitle">
      <property name="text">
       <string>Home</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblSubtitle">
      <property name="text">
       <string>Klik salah satu tombol di bawah untuk membuka tampilan tabel.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line"/>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>14</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="btnOfficers">
        <property name="toolTip">
         <string>Buka tabel Officers</string>
        </property>
        <property name="text">
         <string>Officers</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnVehicles">
        <property name="toolTip">
         <string>Buka tabel Vehicles</string>
        </property>
        <property name="text">
         <string>Vehicles</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnDrives">
        <property name="toolTip">
         <string>Buka tabel Drives</string>
        </property>
        <property name="text">
         <string>Drives</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnCycles">
        <property name="toolTip">
         <string>Buka tabel Cycles</string>
        </property>
        <property name="text">
         <string>Cycles</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnActivities">
        <property name="toolTip">
         <string>Buka tabel Activities</string>
        </property>
        <property name="text">
         <string>Activities</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnQueues">
        <property name="toolTip">
         <string>Buka tabel Queues</string>
        </property>
        <property name="text">
         <string>Queues</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
